doc/en/user/source/extensions/wps-download/rawDownload.rst

```diff
@@ -461,7 +461,10 @@ The supported writing parameters are:
    * ``Deflate``   (Lossless)
    
 
- * ``quality`` : Compression quality for lossy compression (JPEG). Value is in the range [0 : 1] where 0 is for worst quality/higher compression and 1 is for best quality/lower compression
+ * ``quality`` : Compression quality. Value is in the range [0 : 1]
+
+   * for ``JPEG`` lossy compression, 0 is for worst quality/higher compression and 1 is for best quality/lower compression. (default is 1).
+   * for ``Deflate`` lossless compression, input value in the range [0 : 1] is linearly mapped to output deflate level in the range [1 : 9]:  ``(deflate level = 1 + 8 * (quality))``, where level 1 is for best speed and level 9 is for best compression. (default level is 9)
```

Comment:
It's a bit weird since the quality does not really change in the deflate case. I guess it's a by product of how the quality param works at the imageio level? (like, it seems you're documenting existing behavior, and I don't remember recent changes dealing with deflate compression control?)